adblock_social_filters/adblock_social_list.txt

```diff
@@ -1047,18 +1043,18 @@ wispol.eu##.slide_button
 wojas.pl##.sv-fb-0
 wp.tv##.social
 wszystkotujest.pl##.is
-www.focus.pl##.social-icon-box
-www.focus.pl#?#.box-obserwuj-newsletter > h2:-abp-contains(Obserwuj nas)
+www.focus.pl##.box-obserwuj-newsletter > a[href="https://www.instagram.com/focus.pl/"], .box-obserwuj-newsletter > h2:first-of-type, .social-icon-box
```

Comment:
@hawkeye116477 dałeś niepotrzebnie 
2x `.box-obserwuj-newsletter >  `, wystarczy raz

`www.focus.pl##.box-obserwuj-newsletter > a[href="https://www.instagram.com/focus.pl/"], h2:first-of-type, .social-icon-box`

Reply:
@gomoku Dzięki